test/UKHO.ADDS.EFS.FunctionalTests/Utilities/TestExecutionHelper.cs

```diff
@@ -0,0 +1,77 @@
+using System.Text.Json;
+using UKHO.ADDS.Clients.Kiota.SalesCatalogueService.Models;
+using UKHO.ADDS.EFS.Domain.Jobs;
+using UKHO.ADDS.EFS.FunctionalTests.Assertions;
+using UKHO.ADDS.EFS.FunctionalTests.Diagnostics;
+using UKHO.ADDS.EFS.FunctionalTests.Http;
+using UKHO.ADDS.EFS.FunctionalTests.IO;
+
+namespace UKHO.ADDS.EFS.FunctionalTests.Utilities
+{
+    public class TestExecutionHelper
+    {
+        private static async Task CommonTestSteps(string requestId, string zipFileName, string[]? productNames = null, bool assertCallbackTxtFile = false, string batchId = "", string responseContent = "")
```

Comment:
This needs a more descriptive method name, describing what it does.  e.g. AwaitJobCompletionAndAssertOnResults